sdcm/sct_events/health.py

```diff
@@ -104,10 +104,20 @@ def msgfmt(self) -> str:
         return super().msgfmt
 
 
+class PartitionRowsValidationEvent(InformationalEvent):
+    def __init__(self, message: str, severity: Severity = Severity.NORMAL):
```

Comment:
Minor nitpick: I would remove default severity, as we always want to specify it anyway, to make it obvious what is being used and I would also switch the order of parameters, so inherited parameters are first